prime-router/src/test/kotlin/fhirengine/azure/FHIRConverterIntegrationTests.kt

```diff
@@ -158,7 +158,7 @@ class FHIRConverterIntegrationTests {
     @BeforeEach
     fun beforeEach() {
         mockkObject(QueueAccess)
-        every { QueueAccess.sendMessage(any(), any()) } returns Unit
+        every { QueueAccess.sendMessage(any(), any()) } returns ""
```

Comment:
Should this be actually returning something besides empty String now?

Reply:
I think in this case it doesn't actually matter since we only need the return value for reporting purposes if we encounter an exception